### PR TITLE
Add docs for NuGetAuditSuppress

### PR DESCRIPTION
**Pull request by advay26**
Fixes: https://github.com/NuGet/Home/issues/13493

We added the ability to suppress specific advisory warnings during restore with `NuGetAuditSuppress` for PackageReference projects, and will be adding it for packages.config projects soon too.

Design spec: [Suppress specific advisories/CVEs in NuGetAudit](https://github.com/NuGet/Home/blob/dev/accepted/2023/NuGetAudit-supress-advisory.md)

### CODE REVIEW

File: docs/concepts/Auditing-Packages.md [hunk 1]
```diff
@@ -87,8 +87,15 @@ We recommend that audit is configured at a repository level.
 
 ### Excluding advisories
 
-There is no support for excluding individual advisories at this time.
-You can use `<NoWarn>` to suppress `NU1901`-`NU1904` warnings or use the `<NuGetAuditLevel>` functionality to ensure your audit reports are useful to your workflow.
+You can choose to exclude specific advisories from the audit report by adding a new `NuGetAuditSuppress` MSBuild item for each advisory. Define a `NuGetAuditSuppress` item with the `Include=` metadata set to the advisory URL you wish to suppress.
```
Comment (zivkan) [inline docs/concepts/Auditing-Packages.md:90]: one sentence per line is easier to comment on in reviews, and has smaller diffs when modified in the future.

```suggestion
You can choose to exclude specific advisories from the audit report by adding a new `NuGetAuditSuppress` MSBuild item for each advisory.
Define a `NuGetAuditSuppress` item with the `Include=` metadata set to the advisory URL you wish to suppress.
```

File: docs/concepts/Auditing-Packages.md [hunk 2]
```diff
@@ -87,8 +87,15 @@ We recommend that audit is configured at a repository level.
...
+
+```xml
+    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
+```
+
+Similar to the other NuGet audit configuration properties, `NuGetAuditSuppress` items can be defined at the project or repository level.
+
+`NuGetAuditSuppress` is available for PackageReference projects. It is not currently available for packages.config projects.
```
Comment (zivkan) [inline docs/concepts/Auditing-Packages.md:98]: one sentence per line

```suggestion
`NuGetAuditSuppress` is available for PackageReference projects.
It is not currently available for packages.config projects.
```

Comment (zivkan) [inline docs/concepts/Auditing-Packages.md:98]: A lot of customers run old versions of VS, so I think it's good to be clear about versions features are available in. But in this case, the feature is only available in previews, not in GA yet, so I think it's particularly important.

```suggestion
`NuGetAuditSuppress` is available for PackageReference projects starting from NuGet 6.11, Visual Studio 17.11, and the .NET 8.0.400 SDK. It is not currently available for packages.config projects.
```

File: docs/concepts/Auditing-Packages.md [hunk 3]
```diff
@@ -87,8 +87,15 @@ We recommend that audit is configured at a repository level.
 
 ### Excluding advisories
 
-There is no support for excluding individual advisories at this time.
-You can use `<NoWarn>` to suppress `NU1901`-`NU1904` warnings or use the `<NuGetAuditLevel>` functionality to ensure your audit reports are useful to your workflow.
+You can choose to exclude specific advisories from the audit report by adding a new `NuGetAuditSuppress` MSBuild item for each advisory. Define a `NuGetAuditSuppress` item with the `Include=` metadata set to the advisory URL you wish to suppress.
+
+```xml
+    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
```
Comment (zivkan) [inline docs/concepts/Auditing-Packages.md:93]:
```suggestion
<ItemGroup>
    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
</ItemGroup>
```

All other NuGetAudit settings are properties, not items, and I think a lot of customers won't be familiar enough with MSBuild syntax to notice the `Include=` means it's an item.  I think adding the `<ItemGroup>` parent increases the chance customers will be successful.

File: docs/concepts/Auditing-Packages.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (advay26) [reply docs/concepts/Auditing-Packages.md]: I've added this back in at the end of the section:

> Additionally, you have the option to suppress warnings based on their severity.
You can use `<NoWarn>` to suppress `NU1901`-`NU1904` warnings or use the `<NuGetAuditLevel>` functionality to ensure your audit reports are useful to your workflow.

Let me know if I can improve on that

File: docs/concepts/Auditing-Packages.md (hunk 3 above)
Comment (advay26) [reply docs/concepts/Auditing-Packages.md:95]: That's a good point. I've added that in

**Review by zivkan — APPROVED** [review]

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**